ztp/source-crs/extra-manifest/99-sync-time-once-master.yaml

```diff
@@ -0,0 +1,27 @@
+# Automatically generated by extra-manifests-builder
+# Do not make changes directly.
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-sync-time-once-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Sync time once
+            After=network.service
```

Comment:
@vitus133 should we run this `Before=chronyd.service` ?

Reply:
We only run it if chronyd service is disabled:
https://github.com/openshift-kni/cnf-features-deploy/blob/f8ba7b1b77f0924ef87e371b2f428b971db696d1/ztp/extra-manifests-builder/99-sync-time-once/build.sh#L27